README.md

```diff
@@ -1,173 +1,171 @@
 
-### Prerequisites
+## 💻 Getting Started <a name="getting-started"></a>
 
-In order to run this project you need:
+To get a local copy up and running, follow these steps.
 
--Any broswer
+### Setup
 
+Clone this repository to your desired folder:
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+```sh
+  git clone https://github.com/Mutai-Gilbert/Leaderboard.git
+  cd to-do-list
+```
 
-### Setup
+### Install
 
-Clone this repository to your desired folder:
+Install this project with:
 
-- Open the command prompt in the created directory.
+```sh
+  npm install
+```
 
-- On the terminal run this command git clone https://ADD LINK.git
+### Usage
 
-- Go to the repository folder using command prompt cd NAME-OF-FILE
+To run the project in a development server, execute the following command:
 
-- Install the dev dependencies for linters run npm install.
---->
+```sh
+  npm start
+```
 
+To to build for the production, execute the following command:
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+```sh
+  npm run build
+```
 
-<!-- Author -->
 
-👤 **Author**
-Anna Petras
 
-- GitHub: [@annaeliana7](https://github.com/annaeliana7)
+### Deployment
+```sh
+  npm run build
+```
+<a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using [Github Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- AUTHORS -->
+
+👤 **Author1**
 
-## 🔭 Future Features <a name="future-features"></a>
+👤 **Gilbert Mutai**
 
-- Implement some UX improvements: added transitions and/or animation.
+- GitHub: [@githubhandle](https://github.com/Mutai-Gilbert)
+- Twitter: [@twitterhandle](https://twitter.com/@nerdmutai)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mutai-gilbert-2a5a42137/)
 
-- Implement additional pages.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
+<!-- CONTRIBUTING -->
+
 ## 🤝 Contributing <a name="contributing"></a>
 
+ 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://ADD LINK.git).
+Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-Give a ⭐️ if you like this project!
+If you like this project give me a star.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
-💜 HaShem (Abba) and Yeshua, I LOVE YOU! 💜 Thank you!
-I HONOR + GLORIFY YOU FOREVER ... 💜
+
+ I would like to thank all my fellow micronauts.
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
+<!-- LICENSE -->
+
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT] licensed.
```

Comment:
- [ ] Great job working on this milestone. At the moment, your license is not a link. Kindly make it a link and ensure that it does not redirect to an error page.